packages/web/src/app/contexts/general/modals/modals.tsx

```diff
@@ -80,6 +80,10 @@ const modalsMap = new Map<ModalType, LoadableComponent<ModalPropsType & any>>([
     ModalType.SHOW_MODAL_TRANSACTION_STATUS,
     loadable(() => import('components/modals/TransactionInfoModals/TransactionStatusModal')),
   ],
+  [
+    ModalType.SHOW_MODAL_TRANSACTION_STATUS_SWAP,
```

Comment:
You should know that constants have names like files, so this constant should be SHOW_MODAL_TRANSACTION_WAP if file called TransactionSwap